src/lava/magma/runtime/message_infrastructure/examples/ros2/ros_talk_with_dds_py/ros_talk_with_dds_py/publisher.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import rclpy
+from rclpy.node import Node
+from rclpy.qos import qos_profile_system_default
+
+from ddsmetadata.msg import DDSMetaData
+
+
+class MinimalPublisher(Node):
+
+    def __init__(self):
+        super().__init__('minimal_publisher')
+        ros_qos = qos_profile_system_default
+        self.publisher_ = self.create_publisher(DDSMetaData,
+                                                'dds_topic',
+                                                ros_qos)
+        timer_period = 0.5  # seconds
+        self.timer = self.create_timer(timer_period, self.timer_callback)
+        self.i = 0
+        self.msg = DDSMetaData()
```

Comment:
could add a function that converts numpy array to DDSMetaData into `utils.py`?

Reply:
Fix in commit [eca5172](https://github.com/lava-nc/lava/pull/498/commits/eca5172e5d3126b0600e0ec612a80fab0558cf70)